scripts/griddle.jsx

```diff
@@ -707,11 +708,11 @@ var Griddle = React.createClass({
                 hasMorePages={hasMorePages}
                 onRowClick={this.props.onRowClick}/></div>)
     },
-    getContentSection: function(data, cols, meta, pagingContent, hasMorePages){
+    getContentSection: function(data, extraData, cols, meta, pagingContent, hasMorePages){
```

Comment:
Yea that's a great idea to make it more reusable. Thanks again for your time and thinking around this!